networking/hardware_networks/configure-lacp-for-sriov.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="sriov-lacp-sriov"]
+= High availability for pod-level bonds on SR-IOV networks
+include::_attributes/common-attributes.adoc[]
+:context: sriov-lacp-sriov
+
+toc::[]
+
+For workloads using pod-level bonding with SR-IOV virtual functions (VFs), despite an upstream switch failure, an underlying physical function (PF) might still report an `up` state. This creates a silent failure, as attached VFs remain up and pods continue to send traffic to a dead endpoint, causing packet loss.
+
+The PF Status Relay Operator solves this issue by using Link Aggregation Control Protocol (LACP) as an active health check. In this configuration, each physical function (PF) is placed in its own single-member LACP bond with the upstream switch. When the Operator detects an LACP failure on a PF's bond, it propagates this status to the attached VFs. This action triggers the pod's `active-backup` bond to fail over to its backup network path, maintaining high availability.
```

Comment:
Im not sure we should say it propagates the status to the VF. It makes it sounds as if there is some type of control  plane process between the operator and the VF on the pod. Rather is simply changing the status of the VF on the node from auto to disabled. What do you think?